### DISCUSSION TITLE
Binary for CentOS 7.9

### DISCUSSION THREAD

**Discussion by sharkpd** [rustdesk/rustdesk#1040] [Q&A] [answered]
Hello everybody,
is it possible to compile rustDesk for CentOS 7.9?
if yes, can you give me some info to get a usable executable on CentOS 7.9?
the procedure described for the compilation requires higher package versions than those present in CentOS 7.9 ...

Thanks for your help

**Comment by rustdesk** [comment, marked as answer]
Too bad, CentOS 7.9 is too old for us, many dependencies can not be solved on it.
